### ISSUE TITLE
unicycler_polish : samtools sort memory allocation error

### ISSUE THREAD

**Issue by Xanthomonass** [rrwick/Unicycler#143]
Hi there,

I am trying to polish my PacBio assembly with`unicycler_polish` because I'm pretty sure there is a duplication at the start/end of the circular chromosome. I used the PacBio reads in .fasta format as input. However I'm stucked with the following error:
```
`BamPostService returned a non-zero exit status samtools sort --threads 4 -m 4G -o /home/lucas/unicycler_polish/pbalign_alignments.bam /tmp/tmp1PLcn9/fGuIgK.bam. CMD: '1'

ERROR: samtools sort: couldn't allocate memory for bam_mem`
```
I'm running the command on my laptop which has 8 threads and 2*8MB of RAM, so plenty of free memory to use. It may be the same issue as in #139  I attach to this post the entire terminal output in case it will be usefull.

I could really use a hand on this one.

Thanks in advance.

[samtools_memory_error.txt](https://github.com/rrwick/Unicycler/files/2434787/samtools_memory_error.txt)

**Comment by stevenjdunn** [comment]
Have you checked the memory allocation of your virtual box VM?

**Comment by Xanthomonass** [comment]
Yes, I set my VM with 6 threads and 8GB of RAM, which should be enough (except if I misunderstood something).
